Prace_domowe/Praca_domowa2/Mateusz_Ziemła/pd2.ipynb

```diff
@@ -0,0 +1,737 @@
+{
```

Comment:
<p>a dla 100 już absolutnie stabilnie? Dziwne...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>Co to znaczy, że dla 10 jest niestabilne? Możesz wrzucić przykład</p>

Reply:
<p>Porównujesz nie te kolumny</p>

Reply:
<p>Liczysz rmse na całej kolumnie, a nie na imputowanych wartościach</p>